config/module.config.php

```diff
@@ -31,6 +31,11 @@
             'AssetManager\Resolver\AliasPathStackResolver'      => 1000,
             'AssetManager\Resolver\PathStackResolver'           => 500,
         ),
+        'view_helper' => array(
+            'cache'            => null,
+            'query_string'     => '_',
+            'development_mode' => false,
```

Comment:
@pensiero I don't know. I think that maybe there shouldn't be a development flag at all. That's what your dev environment config is for: to change config values based on environment.

I like your suggestion to rename it. What is the behavior this adds? It adds the timestamp, right? Maybe name it something like append_timestamp? Then it becomes a feature in stead of an environment configuration.